sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/models/CopyFileSmbInfo.java

```diff
@@ -126,6 +134,28 @@ public CopyFileSmbInfo setFileLastWriteTime(String fileLastWriteTime) {
         return this;
     }
 
+    /**
+     * Get the fileChangeTime property: Specifies either the option to copy file last write time from a source
+     * file(source) to a target file or a time value in ISO 8601 format to set as last write time on a target file.
+     *
+     * @return the fileChangeTime value.
+     */
+    public String getFileChangeTime() {
+        return this.fileChangeTime;
+    }
+
+    /**
+     * Set the fileChangeTime property: Specifies either the option to copy file last write time from a source
+     * file(source) to a target file or a time value in ISO 8601 format to set as last write time on a target file.
+     *
+     * @param fileChangeTime the fileChangeTime value to set.
+     * @return the CopyFileSmbInfo object itself.
+     */
+    public CopyFileSmbInfo setFileChangeTime(String fileChangeTime) {
```

Comment:
Change Time for File Copy was cut at the last minute - I removed it in .NET here - https://github.com/Azure/azure-sdk-for-net/pull/27565.